user/src/main/java/com/jumar/user/utils/DateUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.jumar.user.utils;
+
+import java.time.LocalDateTime;
+
+public final class DateUtils {
+    private DateUtils() {}
+
+    public static LocalDateTime dateOfBirthFormatter(String dateOfBirth) {
+
```

Comment:
don't need this as 1) it's a LocalDate not LocalDateTime. 2) just parse it in the code or use an external library for dates.